python/plugins/processing/algs/gdal/Buffer.py

```diff
@@ -115,6 +115,10 @@ def getConsoleCommands(self, parameters, context, feedback):
             if f.name() == geometry:
                 continue
             other_fields.append(f.name())
+        if len(other_fields) > 0:
```

Comment:
Should this file be in here?
But I guess that should that read `if other_fields:`

PEP 8:

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.